crates/oxide/src/extractor/pre_processors/clojure.rs

```diff
@@ -26,52 +35,81 @@ impl PreProcessor for Clojure {
                             b'\\' => cursor.advance_twice(),
 
                             // End of the string
-                            b'"' => break,
+                            b'"' => {
+                                result[cursor.pos] = b' ';
+                                break;
+                            }
 
                             // Everything else is valid
                             _ => cursor.advance(),
                         };
                     }
                 }
 
-                // Consume comments as-is until the end of the line.
+                // Discard line comments until the end of the line.
                 // Comments start with `;;`
                 b';' if matches!(cursor.next, b';') => {
                     while cursor.pos < len && cursor.curr != b'\n' {
+                        result[cursor.pos] = b' ';
                         cursor.advance();
                     }
                 }
 
-                // A `.` surrounded by digits is a decimal number, so we don't want to replace it.
-                //
-                // E.g.:
-                // ```
-                // gap-1.5
-                //      ^
-                // ``
-                b'.' if cursor.prev.is_ascii_digit() && cursor.next.is_ascii_digit() => {
+                // Consume keyword until a terminating character is reached.
+                b':' => {
+                    result[cursor.pos] = b' ';
 
-                    // Keep the `.` as-is
-                }
+                    while cursor.pos < len {
+                        cursor.advance();
```

Comment:
Doing this first could raise a potential panic on line 91, move it after the loop.